### DISCUSSION TITLE
Embedding Gate into notes

### DISCUSSION THREAD

**Discussion by Derivedbetter** [nguyenvanduocit/obsidian-open-gate#55] [Q&A]
Whenever I try to use a code block with a gate for any website, I merely get a white screen. Is this only my issue or has something changed with the plugin?
For example, this code: 
![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/93354976/33d23d99-5525-4484-a212-a935fbeb5c81)

Only shows this:

![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/93354976/0511288b-b458-44d8-8a12-958968ccb221)
 and if I change it to this: 
![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/93354976/cdd4146e-7479-4fc8-a466-adec813553bd)
 it shows nothing.

![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/93354976/3d00179c-8026-40ea-8df5-c8357544c8e7)
 
I'm sure it's operator error, but I can't figure it out. Any help would be appreciated.

**Comment by andrewmcgivery** [comment]
Can you provide the gate code you're using as text instead of an image so we can try it out locally? :)

**Reply by Derivedbetter** [reply]
Sure. Just the standard code block with gate, like the tutorial and the repository show. I only get a big white square, no matter the site.

'''gate
  https://windows.com
  height: 400
'''

That's it.  I'm on my phone now, so the tick marks might not be correct, but I use obsidian on my laptop and use code blocks all the time (python, Solidity, c#). I'm sure there's some conflict with another plug-in but I just thought I'd see if the dev had an answer before I did all of that. Actually the code I provided shows nothing, if I put height on the same exact line as the URL, I get a big white box.

**Comment by BrommD** [comment]
hi, 
try the `reuse` option. I've had this issue using MS Loop, when embedding in a notes page, also a white page was shown. By replacing the link in the `gate` with the reuse option, it works.
- i.e. create a profile in Gate plugin, (for me, it is named `Loop`) 
- put this into your note
```` 
    ```gate
   title: Loop
   height: 300
   ``` 
it works for me.
